style.css

```diff
@@ -553,10 +594,121 @@ textarea {
 .primary-text-desktop,
 .ruby,
 .project-title1-desktop,
-.card-image-desktop1 {
+.card-image-desktop1,
+.mobile-menu {
   display: none;
 }
 
+/* Mobile Popup */
+
+.mobile-project-title {
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup-close {
+  width: 16px;
+  height: 16px;
+  margin: 30px 30px 0 26px;
+}
+
+.popup-img {
+  width: 90%;
+  margin: 20px 16px 12px 16px;
+}
+
+hr {
+  width: 90%;
+  margin: auto;
+  background-color: #ebecf0;
+}
+
+.tags-popup {
+  height: 7vh;
+  width: 100%;
+  padding-left: 10px;
+}
+
+/* .button-popup1,
+.button-popup2 {
+  background-color: #fff;
+  color: #396df2;
+  padding: 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 20px 3px;
+  cursor: pointer;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+} */
```

Comment:
- [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face: